pkg/source/markdown.go

```diff
@@ -77,13 +89,21 @@ func (m *markdown) Output() string {
 		s = strings.TrimSpace(s)
 	}
 
+	if !m.OmitNotice() {
+		s = fmt.Sprintf(confluenceEditNotice, m.opts.DoxNoticeFileUrl) + s
+	}
+
```

Comment:
Starts to make you wonder if `OmitNotice` (and `Ignore`?) should go in `opts` at some point?

Reply:
I was originally going to add a new struct to Opts for the notice, which would include a flag for inclusion and the file URL string. Ultimately decided on this approach as the omit-notice directive was defined in this file anyway, so it seemed unnecessary.